src/main/java/de/adesso/anki/battle/transfer/BodyDTO.java

```diff
@@ -2,20 +2,67 @@
 
 import de.adesso.anki.battle.util.Position;
 import de.adesso.anki.battle.world.Body;
+import de.adesso.anki.battle.world.DynamicBody;
+import de.adesso.anki.battle.world.bodies.Vehicle;
+import de.adesso.anki.battle.world.bodies.roadpieces.Roadpiece;
+
+
+import de.adesso.anki.sdk.AnkiGateway;
+import de.adesso.anki.sdk.AnkiVehicle;
+import de.adesso.anki.sdk.messages.*;
 
 public class BodyDTO {
 
     private final Body body;
+    
+    private double speed; 
+    
+    private String BTAdress;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "BTAdress" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S00116](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AS00116))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=32&issues=AWSshItHsk71k7lwCc1e&open=AWSshItHsk71k7lwCc1e)

[ISSUE_KEY]: AWSshItHsk71k7lwCc1e